src/fidesops/schemas/saas/saas_config.py

```diff
@@ -65,10 +64,8 @@ class ClientConfig(BaseModel):
     """Definition for an authenticated base HTTP client"""
 
     protocol: str
-    host: Union[
-        str, ConnectorParamRef
-    ]  # can be defined inline or be a connector_param reference
-    authentication: Strategy
+    host: str
```

Comment:
Changed the host to use the same placeholder pattern (`<host>`) as in other places in the SaaS config.